models/dialogue/policy_0_KerasPolicy/featurizer.json

```diff
@@ -0,0 +1 @@
+{"py/object": "rasa_core.featurizers.MaxHistoryTrackerFeaturizer", "max_history": 5, "remove_duplicates": true, "state_featurizer": {"py/object": "rasa_core.featurizers.BinarySingleStateFeaturizer", "input_state_map": {"intent_goodbye": 0, "intent_greet": 1, "intent_mood_affirm": 2, "intent_mood_deny": 3, "intent_mood_great": 4, "intent_mood_unhappy": 5, "prev_action_deactivate_form": 9, "prev_action_default_ask_affirmation": 11, "prev_action_default_ask_rephrase": 12, "prev_action_default_fallback": 8, "prev_action_listen": 6, "prev_action_restart": 7, "prev_action_revert_fallback_events": 10, "prev_utter_cheer_up": 14, "prev_utter_did_that_help": 15, "prev_utter_goodbye": 17, "prev_utter_greet": 13, "prev_utter_happy": 16}, "num_features": 18, "slot_feature_len": 0, "user_feature_len": 6}, "use_intent_probabilities": false}
```

Comment:
do we need these additional files in `/models` ?

Reply:
no idea, where this is coming from. Just did the rasa train. Will have a look. c

Reply:
deleted the files